tests/mcp_registry/test_discovery.py

```
@@ -273,7 +310,7 @@ def test_validator_cache_is_version_and_dialect_aware(self, monkeypatch):
             registry_helpers.registry_validator.cache_clear()
 
     def test_unbundled_schema_version_is_rejected(self):
-        with pytest.raises(ValueError, match="available versions: 2025-12-11"):
+        with pytest.raises(ValueError, match="2025-07-09.*2025-12-11"):
```

Ruff (0.16.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)